packages/grafana-ui/src/components/index.ts

```diff
@@ -265,6 +265,7 @@ export { type UserView } from './UsersIndicator/types';
 export { InlineFormLabel } from './FormLabel/FormLabel';
 export { Divider } from './Divider/Divider';
 
+/** @deprecated Please use non-legacy versions of these components */
```

Comment:
Where are they?

Reply:
If you go to the file for each of these components, you can see a more detailed deprecation message with the suggested alternatives.